tests/cloner_test.go

```diff
@@ -121,3 +125,30 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func verifyTargetContent(f *framework.Framework, namespace *v1.Namespace, pvc *v1.PersistentVolumeClaim) bool {
+	By("Verify target PVC content matches source PVC")
+	executorPod, err := utils.CreateExecutorPodWithPVC(f.K8sClient, "verify-pvc-content", namespace.Name, pvc)
+	Expect(err).ToNot(HaveOccurred())
+	err = utils.WaitTimeoutForPodReady(f.K8sClient, executorPod.Name, namespace.Name, utils.PodWaitForTime)
+	Expect(err).ToNot(HaveOccurred())
+	output := f.ExecShellInPod(executorPod.Name, namespace.Name, "cat "+testFile)
+	f.DeletePod(executorPod)
+	return strings.Compare(fillData, output) == 0
+}
+
+func cloneOfAnnoExistenceTest(f *framework.Framework, targetNs *v1.Namespace) {
+	// Create targetPvc
+	By("Creating target pvc")
+	_, err := utils.CreatePVCFromDefinition(f.K8sClient, targetNs.Name, utils.NewPVCDefinition(
+		"target-pvc",
+		"1G",
+		map[string]string{controller.AnnCloneRequest: f.Namespace.Name + "/" + sourcePVCName, controller.AnnCloneOf: "true"},
+		nil))
+	Expect(err).ToNot(HaveOccurred())
+	By("Checking no cloning pods were created")
+	_, err = f.FindPodByPrefix(common.CLONER_SOURCE_PODNAME)
+	Expect(err).To(HaveOccurred())
```

Comment:
`apierrs.IsNotFound()` might be a more precise check here (from
`apierrs "k8s.io/apimachinery/pkg/api/errors"`)

Reply:
how do we use it?

Reply:
```
import ( ...
        apierrs "k8s.io/apimachinery/pkg/api/errors"
...

	_, err = f.FindPodByPrefix(common.CLONER_SOURCE_PODNAME)
	Expect(apierrs.IsNotFound(err)).ToBe(True())
	_, err = utils.FindPodByPrefix(f.K8sClient, targetNamespaceName, common.CLONER_TARGET_PODNAME, common.CDI_LABEL_SELECTOR)
	Expect(err).To(HaveOccurred())
```